test/compile/data/density.test.ts

```diff
@@ -54,14 +55,16 @@ describe('compile/data/fold', () => {
       expect(density.assemble()).toEqual({
         type: 'kde',
         field: 'v',
+        resolve: 'shared',
         as: ['A', 'density']
       });
     });
 
-    it('should add resolve shared if we group', () => {
+    it('should only add resolve "shared" if we set it explicitly', () => {
```

Comment:
To test this, we would need a case that doesn't set `resolve` when it's not set in the transform.

Reply:
Won't `resolve` always be in the transform sine the parameter has a default value? I think this test case should just be switched to "independent" instead of "shared" to reflect the updates in e3f4505d06d0f8a6c7e8d1d0900e85c1fdfdb8d0. I did that in my latest commit.